src/pages/Courses/components/FilterTable/FilterTable.jsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+
+import "./FilterTable.css";
+
+const filterData = [
+  {
+    heading: "Category",
+    name: "category",
+    inputType: "checkbox",
+    filterType: [
+      {
+        id: "marketing",
+        label: "Marketing",
+      },
+      {
+        id: "finance",
+        label: "Finance",
+      },
+      {
+        id: "technology",
+        label: "Technology",
+      },
+    ],
+  },
+  {
+    heading: "Ratings",
+    name: "rating",
+    inputType: "radio",
+    filterType: [
+      {
+        id: "four-and-above",
+        label: "4 stars and above",
+      },
+      {
+        id: "three-and-above",
+        label: "3 stars and above",
+      },
+      {
+        id: "two-and-above",
+        label: "2 stars and above",
+      },
+    ],
+  },
+  {
+    heading: "Sort By",
+    name: "sort",
+    inputType: "radio",
+    filterType: [
+      {
+        id: "high-to-low",
+        label: "Price - high to low ",
+      },
+      {
+        id: "low-to-high",
+        label: "Price - low to high",
+      },
+    ],
+  },
+];
+
```

Comment:
You can put this filter data in separate file and export it  from there.

1. It will reduce the line of code in this particular file.

Reply:
Yeah i will move that to data folder